Emby.Server.Implementations/ScheduledTasks/Tasks/ChapterImagesTask.cs

```diff
@@ -161,22 +159,18 @@ public async Task Execute(CancellationToken cancellationToken, IProgress<double>
             }
         }
 
-        /// <summary>
-        /// Gets the name of the task
-        /// </summary>
-        /// <value>The name.</value>
         public string Name => "Chapter image extraction";
 
-        /// <summary>
-        /// Gets the description.
-        /// </summary>
-        /// <value>The description.</value>
         public string Description => "Creates thumbnails for videos that have chapters.";
 
-        /// <summary>
-        /// Gets the category.
-        /// </summary>
-        /// <value>The category.</value>
         public string Category => "Library";
+
+        public string Key => "RefreshChapterImages";
+
+        public bool IsHidden => false;
```

Comment:
(food for thought) do we even _need_ hidden tasks?

Reply:
I agree, I'd rather there be no "hidden" tasks - an administrator should *always* be able to care about what's going on in the background if they want to.